content/working_with/official_plugins/vsphere.md

```diff
@@ -108,6 +108,7 @@ Each type has a `connection_config` property. It can be used to pass parameters
 
 **Properties:**
 
+* `use_existing_resource` - Indicate that the VM has already been created you want to begin using it. Should be used together with the `server:name` property. _Note: Cloudify will not delete or perform any other lifecycle operations aside from monitoring and agent installation if configured._
```

Comment:
@EarthmanT 
\- Indicate that the VM has already been created you want to begin using it.
\+ Indicate that the VM has already been created **and that** you want to begin using it.